src/openzaak/templates/index.html

```diff
@@ -46,7 +46,17 @@ <h5 class="card-title"><i class="fas fa-shield-alt"></i> API scopes</h5>
         {% endblock %}
 
         {% block extra_docs %}
-          {% include "vng_api_common/includes/kanalen_card.html" %}
+            <div class="col-sm-4">
+
+                <div class="card">
+                    <div class="card-body">
+                        <h5 class="card-title"><i class="fas fa-bullhorn"></i> Notificaties</h5>
+                        <p class="card-text">Gebeurtenissen waarover dit component notificaties verstuurt.</p>
+                        <a href="https://github.com/{{ settings.NOTIFICATIONS_REPO }}/blob/{{ settings.NOTIFICATIONS_BRANCH }}/src/notificaties.md" class="btn btn-primary">Notificaties</a>
```

Comment:
This is not the correct link - each API has it's own documentation file, e.g.:

* https://github.com/VNG-Realisatie/zaken-api/blob/stable/1.0.x/src/notificaties.md
* https://github.com/VNG-Realisatie/documenten-api/blob/stable/1.0.x/src/notificaties.md

etc.